vpr/src/base/vpr_types.h

```diff
@@ -1187,7 +1189,7 @@ struct t_trace {
 struct t_rr_node_route_inf {
     int prev_node;
     t_edge_size prev_edge;
-
+    short is_non_configurable;
```

Comment:
This belongs in its own data structure.  Also this is not a bitmap, this is storing 1 bit in a ~16-bit field?

Reply:
Also this data structure is for optimization data, this is a static behavior.

Reply:
The reason for not creating a separate data structure to store the bitmap information is that current t_rr_node_route_inf seems to have 2 bytes free space.  If we plan to change the structure of t_rr_node_route_inf then I can implement a separate bitmap instead.